sensing/autoware_pointcloud_preprocessor/schema/concatenate_and_time_sync_node.schema.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Concatenate and Time Synchronize Node",
+  "type": "object",
+  "definitions": {
+    "concatenate_and_time_sync_node": {
+      "type": "object",
+      "properties": {
+        "maximum_queue_size": {
+          "type": "integer",
+          "default": "5",
+          "description": "Maximum size of the queue for the Keep Last policy in QoS history."
+        },
+        "timeout_sec": {
+          "type": "number",
+          "default": "0.1",
+          "description": "Timer's timeout duration in seconds when collectors are created. Collectors will concatenate the point clouds when timeout_sec reaches zero."
+        },
+        "is_motion_compensated": {
+          "type": "boolean",
+          "default": true,
+          "description": "Flag to indicate if motion compensation is enabled."
+        },
+        "publish_synchronized_pointcloud": {
+          "type": "boolean",
+          "default": true,
+          "description": "Flag to indicate if synchronized point cloud should be published."
+        },
+        "keep_input_frame_in_synchronized_pointcloud": {
+          "type": "boolean",
+          "default": true,
+          "description": "Flag to indicate if input frame should be kept in synchronized point cloud."
+        },
+        "publish_previous_but_late_pointcloud": {
+          "type": "boolean",
+          "default": false,
+          "description": "Flag to indicate if a concatenated point cloud should be published if its timestamp is earlier than the previous published concatenated point cloud."
+        },
+        "synchronized_pointcloud_postfix": {
+          "type": "string",
+          "default": "pointcloud",
+          "description": "Postfix for the topic name of the synchronized point cloud."
+        },
+        "input_twist_topic_type": {
+          "type": "string",
+          "default": "twist",
+          "description": "Type of the input twist topic."
+        },
+        "input_topics": {
+          "type": "array",
+          "items": {
```

Comment:
There is an issue that says that the concatenation does not work with only one lidar.
I think we can make is more clear that concatenation is for two or more lidars by adding a requirement here

It seems that the following could work:
"minItems": 2,